modules/swagger-codegen/src/main/resources/php/api.mustache

```diff
@@ -172,6 +201,33 @@ use \{{invokerPackage}}\ObjectSerializer;
 
         {{#formParams}}// form params
         if (${{paramName}} !== null) {
+            {{#hasValidation}}
```

Comment:
My understanding is that the above check should cover form parameters so there's no need for another set of validation here. Do you agree?